classroom.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'class_student'
+
+class Classroom
+  attr_accessor :label
+  attr_reader :student
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(student)
+    @student.push(student)
```

Comment:
- [ ] I think it would be a great idea to assign the student here to a self like this `student.classroom = self` so each student has its own classroom ✨🙏🏻